ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_path_groups.schema.yml

```
@@ -49,3 +49,17 @@ keys:
               local:
                 type: str
                 description: Optional, if not set, the path-group `name` is used as local.
+        dps_keepalive_interval:
+          type: dict
+          description: |-
+            Period between the transmission of consecutive keepalive messages, and failure threshold.
+          "$ref": "eos_cli_config_gen#/keys/router_path_selection/keys/path_groups/items/keys/keepalive/"
+          keys:
+            failure_threshold:
+              type: int
+              min: 2
+              max: 100
+              default: 5
+              description: Failure threshold in number of intervals. Required when `interval` is set.
```

It is not required to be set, since we have a default value.
```suggestion
              description: Failure threshold in number of lost keep-alive messages.
```

---

Hmm, intention was to convey that it is `only` required if interval is set. Probably should have used `only`.
Rephrased it to what you suggested .